packages/web/src/alliance/Main.tsx

```diff
@@ -0,0 +1,163 @@
+          </PinWheel>
+        </View>
+        <GridRow
+          desktopStyle={standardStyles.sectionMarginBottom}
+          tabletStyle={standardStyles.sectionMarginBottomTablet}
+          mobileStyle={standardStyles.sectionMarginBottomMobile}
+          allStyle={standardStyles.centered}
+        >
+          <Cell span={Spans.half}>
+            <Fade>
+              <CollectiveMission />
+              <SignupForm />
+            </Fade>
+          </Cell>
+        </GridRow>
+      </View>
+      <Affirmations />
+      <BeautifulQuote
+        color={colors.white}
+        imgSource={aguaComunidad}
+        quote={'"Viva La Local"'}
+        citation={'who said it'}
+      />
+      <Members />
+      <BeautifulQuote
+        color={colors.dark}
+        imgSource={fences}
+        quote={'"Just Do It"'}
+        citation={'Nike'}
+      />
+      <ArticleData title={t('mediumArticlesTitle')} />
+      <Benefits />
+      <WheelWithForm />
+      <ConnectionFooter includeDividerLine={false} />
+    </View>
+  )
+}
+
+const keyframes = [
+  {
+    '0%': {
+      opacity: 0.1,
+      background: `linear-gradient(90deg, rgba(12,218,110,1) 21%, rgba(255,101,83,1) 40%, rgba(159,105,255,1) 60%, rgba(82,182,255,1) 80%)`,
+    },
+    '15%': {
+      background: `linear-gradient(90deg, rgba(82,182,255,1) 18%, rgba(12,218,110,1) 39%, rgba(255,101,83,1) 63%, rgba(159,105,255,1) 80%)`,
+    },
+    '30%': {
+      background: `linear-gradient(90deg, rgba(159,105,255,1) 22%, rgba(82,182,255,1) 38%, rgba(12,218,110,1) 62%, rgba(255,101,83,1) 81%)`,
+    },
+    '45%': {
+      background: `linear-gradient(90deg, rgba(255,101,83,1) 20%, rgba(159,105,255,1) 40%, rgba(82,182,255,1) 62%, rgba(12,218,110,1) 84%)`,
+    },
+    '60%': {
+      background: `linear-gradient(90deg, rgba(12,218,110,1) 21%, rgba(255,101,83,1) 40%, rgba(159,105,255,1) 60%, rgba(82,182,255,1) 80%)`,
+    },
+    '75%': {
+      background: `linear-gradient(90deg, rgba(255,255,255,1) 0%, rgba(12,218,110,1) 23%, rgba(255,101,83,1) 44%, rgba(159,105,255,1) 68%, rgba(82,182,255,1) 90%)`,
+    },
+    '80%': {
+      background: `linear-gradient(90deg, rgba(255,255,255,1) 22%, rgba(12,218,110,1) 43%, rgba(255,101,83,1) 66%, rgba(159,105,255,1) 88%, rgba(82,182,255,1) 100%)`,
+    },
+    '85%': {
+      background: `linear-gradient(90deg, rgba(255,255,255,1) 42%, rgba(12,218,110,1) 65%, rgba(255,101,83,1) 87%, rgba(159,105,255,1) 100%, rgba(82,182,255,1) 100%)`,
+    },
+    '90%': {
+      background: `linear-gradient(90deg, rgba(255,255,255,1) 64%, rgba(12,218,110,1) 81%, rgba(255,101,83,1) 98%, rgba(159,105,255,1) 100%, rgba(82,182,255,1) 100%)`,
+    },
+    '95%': {
+      background: `linear-gradient(90deg, rgba(255,255,255,1) 86%, rgba(12,218,110,1) 97%, rgba(255,101,83,1) 98%, rgba(159,105,255,1) 100%, rgba(82,182,255,1) 100%)`,
+    },
+    '100%': {
+      opacity: 0.9,
+      backgroundColor: colors.white,
+    },
+  },
+]
+
+const blendStyle = { mixBlendMode: 'darken' }
```

Comment:
why is this out here?

Reply:
oddly whenever blendMode is part of Stylesheet.create it gets lost